File: question2.java
import java.util.Scanner;

public class question2 {
    public static void calculateFactorial(int n) {
        long fact = 1;
        for (int i = 1; i <= n; i++) {
            fact *= i;
        }
        System.out.println("Factorial: " + fact);
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int num = sc.nextInt();
        calculateFactorial(num);
        sc.close();
    }
}

File: question_2.c
#include<conio.h>
#include<stdio.h>
void main(){
    int n,i,f=1;
    printf("Enter the Number");
    scanf("%d",&n);
    for(i=1;i<=n;i++){
        f=f*i;
    }
    printf("the factorial of %d is the %d",n,f);

}